src/modules/commander/Commander.hpp

```diff
@@ -101,6 +93,32 @@ class Commander : public ModuleBase<Commander>, public ModuleParams
 
 	void mission_init();
 
+	/**
+	 * Update the telemetry status and the corresponding status variables.
+	 * Perform system checks when new telemetry link connected.
+	 */
+	void poll_telemetry_status(bool checkAirspeed, bool *hotplug_timeout);
```

Comment:
Passing airspeed to telemetry status poll is pretty odd. I understand why it was temporarily necessary though. I hope to get that under control in commander soon.

Reply:
I agree, but currently it is not avoidable and fixing it should be done in another PR in my opinion.